packages/iota/src/models/payloads/ITaggedDataPayload.ts

```diff
@@ -15,10 +15,10 @@ export interface ITaggedDataPayload extends ITypeBase<5> {
     /**
      * The tag to use to categorize the data.
      */
-    tag: HexEncodedString;
+    tag?: HexEncodedString;
```

Comment:
Why did we make these fields optional ? The change doesn't seam in scope for validation.